From 7311e6fcad56201562528f7fbaadbc2b3f9153f6 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Tue, 5 Jul 2016 14:33:11 +0800
Subject: [PATCH] arm64: dts: rockchip: set swiotlb buffer size on rk3399
 android board

We don't use swiotlb, so keep the size as small as possible.

Change-Id: I1377f4b31112902a5841eafde094d2d12f48e97c
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index 1a4f919a4212..f5e31302ea80 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -51,7 +51,7 @@
 	};
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff1a0000";
+		bootargs = "earlycon=uart8250,mmio32,0xff1a0000 swiotlb=1";
 	};
 
 	ramoops_mem: ramoops_mem {
-- 
2.35.3

